### ISSUE TITLE
This benchmark could lead to wrong conclusion.

### ISSUE THREAD

**Issue by dannyhung1128** [AILab-CVC/SEED-Bench#7]
Example false evaluation:
```
'question_id': '20128',
'q_type_id': 2, 
'question': 'What is visible in the image besides the sea?'
'gt': 'Building'
'prediction': 'Trees'
'open_prediction': In the image, besides the sea, there are several buildings and structures visible, \
                                including a pier, a lighthouse, and a pier building.
'match': False
```
The model's answer was correct as shown in `open_prediction`. However, the metrics said otherwise. 

---- update
After checking on around 120 examples, I found that **the false evaluation rate is above 50%.** There can be either false positive or false negative matches.

False negative match:
```
'question_id': '89257'
'q_type_id': 1
'question': 'What is the man in the suit doing in the image?'
'gt': 'Taking a picture'
'prediction': 'Standing with his arms crossed'
'open_prediction': 'The man in the suit is posing with a woman in a dress for a photo.'
'match': False
```

False positive match:
```
'question_id': '34143'
'q_type_id': 5
'question': 'How many people are present in the image?'
'gt': 'Four'
'prediction': 'Four'
'open_prediction': '2 people'
'match': True
```

The model used for the above test is finetuned `LLaVA-vicuna-7B`

**Comment by Bohao-Lee** [comment]
We adopt the answer ranking strategy [1, 2, 3] for evaluating existing MLLMs with multiple-choice questions. Specifically, for each choice of a question, we compute the likelihood that an MLLM generates the content of this choice given the question. We select the choice with the highest likelihood as model’s prediction. Our evaluation strategy does not rely on the instruction-following capabilities of models to output “A” or “B” or “C” or “D”.

[1] Dai, Wenliang, et al. "InstructBLIP: Towards General-purpose Vision-Language Models with Instruction Tuning." *arXiv preprint arXiv:2305.06500* (2023). [https://arxiv.org/abs/2305.06500](https://arxiv.org/abs/2305.06500)
[2] Brown, Tom, et al. "Language models are few-shot learners." *Advances in Neural Information Processing Systems* 33 (2020): 1877-1901.
[3] Lin, Stephanie, Jack Hilton, and Owain Evans. "Truthfulqa: Measuring how models mimic human falsehoods." *arXiv preprint arXiv:2109.07958* (2021). [https://arxiv.org/abs/2109.07958](https://arxiv.org/abs/2109.07958)

**Comment by dannyhung1128** [comment]
I understand, and am simply pointing out the fundamental flaws using likelihood as an evaluation index: this may not be the best approach to evaluate a LLM.

**Comment by geyuying** [comment]
Thank you for sharing your findings. With multiple-choice questions, our benchmark aims to provide an objective and effective evaluation of Multimodal LLMs, since it is difficult to evaluate open-form prediction objectively. However, currently, considering that some Multimodal LLMs lack the instruction-following capabilities to output “A” or “B” or “C” or “D”, we follow previous work to adopt the answer ranking strategy for a fair comparison. Since LLaVA-vicuna-7B is fine-tuned on multimodal data with relatively long texts,  it may not be good at responding to short answers in our choices. We haven't found similar observations in other models as you point out.
